cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -196,6 +196,12 @@ private predicate isInvalidFunction(Function func) {
     expr.getEnclosingFunction() = func and
     not exists(expr.getType())
   )
+  or
+  exists(Location l1, Location l2 |
```

Comment:
I am curious why we can't do `strictcount(func.getEntryPoint()) > 1`.

Reply:
Then we lose results in traced builds, because we in some exceptional circumstances extract some functions twice.